test/simplelabel/touchstone/traindir/log/likelihood.tab

```diff
@@ -0,0 +1,7 @@
+-8.71589076364230459e+03
```

Comment:
These files match exactly locally and on the test machine? I'm surprised there isn't some need for some regex on the last 4+ digits here.

Reply:
I didn't implement this with Regex because these files match locally and on the test machine, and axisting tests do not generally use regex. While there are a few instances of Regex, most files use Regex on at most a few values. I will bring this up at the meeting, as it might be wise to switch all values to Regex.